python/cuml/common/array_sparse.py

```diff
@@ -15,18 +15,18 @@
 #
 import cupyx as cpx
 import numpy as np
-
 from cuml.common.import_utils import has_scipy
-from cuml.common.input_utils import input_to_cuml_array
+from cuml.common.memory_utils import class_with_cupy_rmm
 from cuml.common.logger import debug
-from cuml.common.memory_utils import with_cupy_rmm
 
+import cuml.common
```

Comment:
This is necessary to avoid a circular import. `cuml.common.input_utils` needs to import `SparseCumlArray` for many of its utility functions. This was the only way to prevent import errors.